dags/ped_data_pub.py

```diff
@@ -0,0 +1,59 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Scrapes trail counter data from the public eco-counter website and publishes it in Socrata",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+}
+
+one_pass_entry_name = "Ped Data Pub"
```

Comment:
I think it would be worth considering giving this entry a more verbose name. Maybe something like "Trail Counter Publisher (ped_data_pub.py)"

Reply:
@Charlie-Henry we have a separate entry in the 1pass vault for socrata api access. i suggest either using that, or extracting the socrata username/pw from this entry and creating a separate entry for it.